docs/web/.vitepress/config.mts

```diff
@@ -87,7 +87,7 @@ export default defineConfig({
     },
     socialLinks: [{ icon: "github", link: "https://github.com/receptron/graphai" }],
     editLink: {
-      pattern: "https://github.com/receptron/website/edit/main/docs/:path",
+      pattern: "https://github.com/receptron/graphai/edit/main/docs/:path",
```

Comment:
This is the link to the image in the document. You can click it to directly access the code editing page on GitHub.

<img width="377" alt="image" src="https://github.com/user-attachments/assets/519f6a9a-9874-413a-9b82-ef69c5f2c66f" />